index.html

```diff
@@ -206,6 +206,20 @@ <h2>Skills</h2>
 
     </div>
   </section>
+
+  <section class="form-background">
+    <h1 class="form-heading">Contact me</h1>
```

Comment:
- [ ] Please make sure that there's only one `h1` tag on your page and then the following header would follow a hierarchy. This is considered good practice and improves the SEO of your website.